### ISSUE TITLE
[Request] >>autoplaylist (playlist link)

### ISSUE THREAD

**Issue by bknie1** [aurora-pro/apex-sigma#486]
Instead of defaulting to someone else's autoplaylist I think this would be a great way to kick off a queue. I know RhinoBot (MusicBot) allows this feature. Would it be possible to accept an optional parameter? The default function would refer to the autoplaylist that is used now, but if a parameter is passed, generate the autoplaylist using the passed playlist link.

**Comment by AXAz0r** [comment]
The current autoplaylist command uses Sigma's specialized playlist file in /music/res names autoplaylist.txt.
I MIGHT allow custom playlist text files to be uploaded with a collection of links.
The reason loading youtube playlists is disabled is due to the long parsing time of them, same reason why autoplaylist items are without precise information.
Until i design a system that can gradually grab the information it needs, this is a no-go.

tl;dr:
playlist text files - maybe
youtube playlist links - not soon

**Comment by bknie1** [comment]
Cool, thanks!
